SQL/0000-00-01-Permission.sql

```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
+    (4,'context_help','Help Module Edit Documentation','2'),
     (5,'bvl_feedback','Behavioural QC','1'),
-    (6,'imaging_browser_qc','Edit imaging browser QC status','2'),
-    (7,'mri_efax','Edit MRI Efax files','2'),
-    (8,'send_to_dcc','Send to DCC','2'),
-    (9,'unsend_to_dcc','Reverse Send from DCC','2'),
-    (10,'access_all_profiles','Across all sites access candidate profiles','2'),
+    (6,'imaging_browser_qc','Imaging Browser Module Edit QC Status','2'),
+    (8,'send_to_dcc','Candidate Profile Module Send to DCC All Sites','2'),
+    (9,'unsend_to_dcc','Candidate Profile Module Reverse Send from DCC own Site','2'),
+    (10,'access_all_profiles','Candidate Profile Access All Sites','2'),
     (11,'data_entry','Data entry','1'),
-    (12,'examiner_view','Add and certify examiners','2'),
-    (13,'examiner_multisite','Across all sites add and certify examiners','2'),
-    (14,'training','View and complete training','2'),
-    (15,'timepoint_flag','Edit exclusion flags','2'),
-    (16,'timepoint_flag_evaluate','Evaluate overall exclusionary criteria for the timepoint','2'),
-    (17,'conflict_resolver','Resolving conflicts','2'),
-    (18,'data_dict_view','View Data Dictionary (Parameter type descriptions)','2'),
-    (19,'violated_scans_view_allsites','Violated Scans: View all-sites Violated Scans','2'),
-    (22,'config','Edit configuration settings','2'),
-    (23,'imaging_browser_view_site','View own-site Imaging Browser pages','2'),
-    (24,'imaging_browser_view_allsites', 'View all-sites Imaging Browser pages', '2'),
-    (25,'dicom_archive_view_allsites', 'Across all sites view Dicom Archive module and pages', '2'),
-    (28,'instrument_builder', 'Instrument Builder: Create and Edit instrument forms', '2'),
-    (29,'data_dict_edit','Edit Data Dictionary','2'),
-    (31,'candidate_parameter_view','View Candidate Parameters','2'),
-    (32,'candidate_parameter_edit','Edit Candidate Parameters','2'),
-    (33,'genomic_browser_view_site','View Genomic Browser data from own site','2'),
-    (34,'genomic_browser_view_allsites','View Genomic Browser data across all sites','2'),
-    (35,'document_repository_view','Document Repository: View','2'),
-    (36,'document_repository_delete','Document Repository: Delete','2'),
-    (37,'server_processes_manager','View and manage server processes','2'),
-    (38,'imaging_uploader','Imaging Uploader','2'),
-    (39,'acknowledgements_view','View Acknowledgements','2'),
-    (40,'acknowledgements_edit','Edit Acknowledgements','2'),
-    (41,'dataquery_view','View Data Query Tool','2'),
-    (42,'genomic_data_manager','Manage the genomic files','2'),
-    (43,'media_write','Media files: Uploading/Downloading/Editing','2'),
+    (12,'examiner_view','Examiner Module Edit Own Site','2'),
+    (13,'examiner_multisite','Examiner Module Edit All Sites','2'),
+    (15,'timepoint_flag','Candidate Parameter Module Edit Exclusion Flags Own Site','2'),
```

Comment:
This permission does not seem to be used anywhere

@driusan confirm ?

Reply:
Yes, I think that was an NIHPD thing.